draft-ietf-tls-tls13.md

```diff
@@ -2105,7 +2084,7 @@ client's "pre_shared_key" extension. In addition, it MUST verify that
 the following values are consistent with those negotiated in the
 connection during which the ticket was established.
 
-- The TLS version number, symmetric ciphersuite, and the hash for HKDF.
+- The TLS version number, symmetric cipher suite, and the hash for HKDF.
```

Comment:
HDKF hash is implicit in the cipher suite now, if I'm reading the definition correctly.